src/commands/Minion/groupkill.ts

```diff
@@ -60,18 +60,21 @@ export default class extends BotCommand {
 	@minionNotBusy
 	@requiresMinion
 	@ironsCantUse
-	async run(msg: KlasaMessage, [inputQuantity, monsterName]: [number | undefined, string]) {
+	async run(msg: KlasaMessage, [monsterName, maximumSizeForParty]: [string, number]) {
 		const monster = findMonster(monsterName);
 		if (!monster) throw `That monster doesn't exist!`;
 		if (!monster.groupKillable) throw `This monster can't be killed in groups!`;
 
 		this.checkReqs([msg.author], monster, 2);
 
+		let maximumSize = 50;
+		if (maximumSizeForParty) maximumSize = maximumSizeForParty;
```

Comment:
this would be a lot simpler by just doing: `(maxSize ?? maxPartySize) - 1`  below I think.